test/api/workflows_format_2/converter.py

```diff
@@ -1,14 +1,13 @@
-import os
-import sys
+"""Functionality for converting a Format 2 workflow into a standard Galaxy workflow."""
+from __future__ import print_function
 
-import yaml
+from collections import OrderedDict
```

Comment:
I swapped the flake8-import-order from ``cryptography`` to ``google`` - I prefer  ``cryptography`` myself but I thought people would like to see the word ``google``. Can we agree as a project to  ``cryptography``?